flake.nix

```diff
@@ -42,6 +42,13 @@
     treefmt-nix.url = "github:numtide/treefmt-nix";
     treefmt-nix.inputs.nixpkgs.follows = "nixpkgs";
 
+    # Pict-rs is the image hosting server for Lemmy.
+    # 0.4 comes with a separation of database and file storage which we require but isn't
+    # in a stable release yet.
+    # After pict-rs 0.4 is in nixpkgs we can remove this input.
+    pict-rs.url = "git+https://git.asonix.dog/asonix/pict-rs.git";
```

Comment:
Is this safe to be updated twice a week or should it be pinned to a known good commit?

Reply:
I've pinned it to `0.4.0-rc11`. While I think it's _probably_ safe I don't want to stick any other admin with debugging this.